boards/arm/contextualelectronics_abc/doc/index.rst

```diff
@@ -0,0 +1,155 @@
+.. _contextualelectronics_abc:
+
+Contextual Eletronics Advanced BLE Cell
+#######################################
+
+Overview
+********
+
+The Contextual Eletronics ABC (PCA10056) hardware provides support for the
+Nordic Semiconductor nRF52840 ARM Cortex-M4F CPU and the following devices:
+
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* Quectel BG95 Modem
+
+.. figure:: img/contextualelectronics_abc.jpg
+     :width: 2046px
+     :align: center
+     :alt: Contextual Electronics Advanced BLE Cell
+
+     Contextual Electronics Advanced BLE Cell (Credit: Chris Gamell)
+
+More information about the board can be found at the `ABC Board website`_.
+The `Nordic Semiconductor Infocenter`_ contains the processor's information
+and the datasheet.
+
+Hardware
+********
+
+ABC board has two external oscillators. The frequency of the slow clock
+is 32.768 kHz. The frequency of the main clock is 32 MHz.
+
+Supported Features
+==================
+
+The contextualelectronics_abc board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M)    | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| Modem     | on-board   | quectel_bg9x         |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `ABC Board website`_ for more details on this board, and
```

Comment:
It's not obvious from  https://contextualelectronics.com/ where this board can be found; a link to https://contextualelectronics.com/courses/advanced-ble-cell/ might be more helpful.

Reply:
Fixed !